src/style.css

```diff
@@ -1,3 +1,67 @@
 body {
   margin: 0;
+  font-family: Roboto, Helvetica, sans-serif;
+}
+
+.logo__img {
+  padding: 10px 0;
+  display: block;
```

Comment:
`padding: 10px 0;` is redundant here, because you `.logo` is centered by the parent's `align-items: center`